src/editor/index.js

```diff
@@ -85,6 +86,11 @@ export default class Editor {
       if (tooltip && mouseEventNearPosition(e, editorElement, tooltip.marker.getStartScreenPosition(), tooltip.element.offsetWidth, tooltip.element.offsetHeight)) {
         return
       }
+      // NOTE: Ignore if file is too big
+      if (editorBuffer.cachedText.length > (300 * 1024)) {
```

Comment:
Please use `TextBuffer::getMaxCharacterIndex()` instead of a non-public attribute.